### PR TITLE
Fix publishing and edit bugs

### PR DESCRIPTION
**Pull request by NatalieEO**
**Description**
https://ucsc-cgl.atlassian.net/browse/SEAB-3826
Don't need to refresh page after publishing/unpublishing tool to see list updated

https://ucsc-cgl.atlassian.net/browse/SEAB-3844
Change text in edit version

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [ ] Check that your code compiles by running `npm run build`
- [ ] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [ ] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [ ] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [ ] Do not use cookies, although this may change in the future
- [ ] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [ ] Do due diligence on new 3rd party libraries, checking for CVEs
- [ ] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

File: src/app/workflow/version-modal/version-modal.component.html [hunk 1]
```diff
@@ -40,7 +38,7 @@ <h4 mat-dialog-title>
           </div>
           <div class="form-group form-group-sm">
             <label class="col-sm-3 col-md-3 col-lg-3 control-label">
-              {{ (isService$ | async) ? 'Service Path:' : 'Workflow Path:' }}
+              {{ entryTypeText + ' Path' }}
```
Comment (coverbeck) [inline src/app/workflow/version-modal/version-modal.component.html:41]: I think you lost the colon in changing this. 

I actually think we should get rid of the colons for labels, but that should be across the board. :)

**Review by coverbeck — APPROVED** [review]

File: src/app/workflow/version-modal/version-modal.component.ts [hunk 2]
```diff
@@ -82,6 +85,15 @@ export class VersionModalComponent implements OnInit, AfterViewChecked, OnDestro
 
   ngOnInit() {
     this.isService$ = this.sessionQuery.isService$.pipe(shareReplay());
+    this.sessionQuery.entryType$.pipe().subscribe((type: EntryType) => {
```
Comment (garyluu) [inline src/app/workflow/version-modal/version-modal.component.ts:88]: empty pipe, missing unsubscribe, probably switch case might look better

**Review by denis-yuen — APPROVED** [review]

**Review by kathy-t — COMMENTED** [review]
Came across this while creating a GitHub App tool (no other tools registered). I checked this branch out locally and the tool looked like this:

Screenshot 2022-02-02 at 09-29-47 Dockstore My Tools

**Comment by NatalieEO** [comment]
> Came across this while creating a GitHub App tool (no other tools registered). I checked this branch out locally and the tool looked like this:

Just fixed this, but please check again locally

**Review by garyluu — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Pull request merged**